src/util/cmdlineargs.cpp

```diff
@@ -133,3 +136,23 @@ void CmdlineArgs::printUsage() {
 
     fputs("\n\n(For more information, see http://mixxx.org/wiki/doku.php/command_line_options)\n",stdout);
 }
+
+bool CmdlineArgs::getSettingsPathStatus() const {
+
+    QString path = getSettingsPath();
+    //info is neccessary as, only making of file doesn't tell about readablilty
+    QFileInfo info(path); 
+    bool status = true;
+    QTemporaryFile fle(path + "/temp_file");
+    if(fle.open()) {
+        fle.close();
+    }
+    else {
+        status = false;
+    }
+    if(!info.isReadable()) {
```

Comment:
Is this required? Can we create a file above without the dir is beeing readable?